druid/src/widget/flex.rs

```diff
@@ -195,8 +195,8 @@ pub struct FlexParams {
     alignment: Option<CrossAxisAlignment>,
 }
 
-#[derive(Clone, Copy)]
-pub(crate) enum Axis {
+#[derive(Data, Debug, Clone, Copy, PartialEq)]
+pub enum Axis {
```

Comment:
Doc comments here, please, and also elsewhere in this file. Also, if this isn't just used for Flex, maybe it makes sense to put it elsewhere?

Reply:
I mentioned that in the PR description - and I agree... not sure where exactly it should go though.

Reply:
I still do not know where to put it. 
I think it should be dealt with in a subsequent PR by someone who does.